source/deploy/consider-mobile-vpn-options.rst

```diff
@@ -20,7 +20,7 @@ A Virtual Private Network (VPN) allows a device outside a firewall to access con
 
 We recommend one of two options: `per-app VPN <#id3>`_ or a `device VPN <#id4>`_ to secure your deployment. Both options are compatible with most EMM providers. 
 
-We also recommend following our `recommended steps to secure your deployment <https://docs.mattermost.com/mobile/mobile-appstore-install.html>`__ and to review the following commonly-asked questions about data security on mobile devices:
+We also recommend following our `recommended steps to secure your deployment <https://docs.mattermost.com/deploy/consider-mobile-vpn-options.html>`__ and to review the following commonly-asked questions about data security on mobile devices:
```

Comment:
Thanks for the update! Could you let us know why this link was chosen to replace the previous on? The linked page doesn't seem to have any steps in it so I am wondering if there may be a better option than this link. :)

Reply:
Just wanted to follow up on this question as it would be great to confirm the replacement does contain the correct information so we don't merge inaccurate content.